Sources/Run/main.swift

```diff
@@ -1,3 +1,9 @@
 import App
+import Vapor
 
-try app(.detect()).run()
```

Comment:
Things done here shouldn't be in `app.swift`, `boot.swift` and `configure.swift` instead?

Reply:
Bootstrapping the logging system and creating and running the app must be done in `main.swift`